README.md

```diff
@@ -9,6 +9,11 @@ This module's objective is to provide specific implementation extensions. To sta
 Description
 -----------
 
+Configuration
```

Comment:
There are 2 properties 1)Export Anonymise Config Path 2)Export Files Directory you have covered only one. Also group this "Configuration" under "FHIR Export"